coorx/nonlinear.py

```diff
@@ -106,7 +108,7 @@ def _map(self, coords):
 
     def _imap(self, coords):
         ret = np.empty(coords.shape, coords.dtype)
-        ret[..., 0] = np.arctan2(coords[..., 0], coords[..., 1])
+        ret[..., 0] = np.arctan2(coords[..., 1], coords[..., 0])  # arctan2(y, x) for correct quadrant
```

Comment:
this line worries me. can you prove this is correct?

Reply:
the round-trip test passes, but I'd like to see smaller fractions of pi represented in that test before I trust this.